### PR TITLE
geoseries events added

### PR DESCRIPTION
**Pull request by iamtekson**
## What does this PR do? 

- Added the geoseries event details!
- Added @AayushChand1 name on the poster contribution

## Consideration 

N/A

## How to test? 

will be available after publish in a github page.

### CODE REVIEW

File: docs/Events/2024_05_24_geoseries_14.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+### Short Description
+
+Rohit Gautam specializes in geospatial software solutions integrating maps, location data, and advanced analytics. He has contributed to popular open-source mapping projects like Leafmap and created OsmGPT, a tool that uses AI to retrieve data from OpenStreetMap through plain language queries. He explores new applications for large language models and generative AI by combining mapping with advanced AI technologies.
+
+### Video Recording
+
+https://youtu.be/7bO8-1sL0hE?si=_Z0BS_AMPBiYmYgo
+
+### Poster
+
```
Comment (iamtekson) [inline docs/Events/2024_05_24_geoseries_14.md:24]: @rowheat02, can you please share your slides? The better approach might be, upload it to the slideshare site and add the URL here.

File: docs/Events/2024_06_07_geoseries_15.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+**Affiliation:** Geospatial Developer, NAXA
+
+### Short Description
+
+Tek Kshetri is a passionate Geospatial expert with over 8 years of hands-on experience in the field. He is a strong advocate for open source, open standards, and open solutions. Driven by his passion for open source, Tek co-founded the OSGeo local chapter in Nepal and actively supports OSGeo's vision and mission.
+
+His primary research interests lie at the intersection of Artificial Intelligence and the Geospatial domain, including natural disaster modelling, Web GIS, and satellite imagery analysis using ML/DL. Tek has developed several open-source packages such as geoserver-rest, GeoTile, pySLD, and leaflet-geoserver-request etc. He also shares his knowledge and insights through his YouTube channel, GeoDev.
+
+### Video Recording
+
```
Comment (iamtekson) [inline docs/Events/2024_06_07_geoseries_15.md:22]: Other details will be soon updated after the event.

File: docs/Events/2024_06_07_geoseries_15.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+### Poster
+
```
Comment (iamtekson) [inline docs/Events/2024_06_07_geoseries_15.md:24]: Updated once it is ready.

**Review by iamtekson — COMMENTED** [review]
@kshitij-raj-sharma, @varun2948 please review it.

**Review by varun2948 — APPROVED** [review]

**Review by kshitijrajsharma — APPROVED** [review]

File: docs/Events/2024_05_24_geoseries_14.md (hunk 1 above)
Comment (iamtekson) [reply docs/Events/2024_05_24_geoseries_14.md:24]: Thank you Rohit.

**Comment by iamtekson** [comment]
I will resolve this conflict after filling all the required details.

**Pull request merged**